### PR TITLE
Added paparazzi screen test (#2923)

### LINKED ISSUE #2923: Setup Paparazzi screenshot tests
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.

### What would you like to improve?

Setup Paparazzi
https://github.com/cashapp/paparazzi

### Because...

Protects us from UI regressions like #2903 and also will catch runtime crashes in the Compose UI.

### Description

Setup as expected and execute in the CI. Make a new workflow that's called "Paparazzi screenshot tests"
https://github.com/cashapp/paparazzi

### Success Criteria

- Git LFS setup
- Proper Paparazzi setup as described in https://github.com/cashapp/paparazzi
- Being able to repord and verify Paparazzi
- CI workflow that run the paparazzi tests

### PR DESCRIPTION
**Pull request by Vivekban**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [x] I confirm that I've run the code locally and everything works as expected.
- [ ] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?
1. Added screenshot test to avoid regressions

Describe with a few bullets **what's new:**
- Added tests for the reports module

> 💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!

## Risk Factors

**What may go wrong if we merge your PR?**
- There may be a failure of the paparazzi test in the future due to dynamic data like dates.

**In what cases your code won't work?**
- There may be a failure of the paparazzi test in the future due to dynamic data like dates. Check troubleshooting guidelines in that case.

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #2923

> Replace `ISSUE_NUMBER` with the number of the GitHub issue that this PR is fixing. If you've done that correctly, you'll see the issue title linked when previewing your PR description.

> **!Note:** Do **not** link the PR number. Link the number/id of the GitHub Issue from [issues](https://github.com/Ivy-Apps/ivy-wallet/issues).

## Troubleshooting CI failures ❌

GitHub Actions failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

File: screen/settings/src/test/snapshots/images/com.ivy.settings_SettingsPaparazziTest_snapshot Home Screen[Dark].png
Comment (ILIYANGERMANOV) [inline screen/settings/src/test/snapshots/images/com.ivy.settings_SettingsPaparazziTest_snapshot Home Screen[Dark].png:1]: The Home snapshots are broken

Comment (Vivekban) [reply screen/settings/src/test/snapshots/images/com.ivy.settings_SettingsPaparazziTest_snapshot Home Screen[Dark].png:1]: fixed issue

**Comment by Vivekban** [comment]
@ILIYANGERMANOV This PR contains fix for test failure for Home

**Review by ILIYANGERMANOV — APPROVED** [review]
Nice 💯

**Pull request merged**